docx/enum/header.py

```diff
@@ -0,0 +1,39 @@
+# encoding: utf-8
+
+from __future__ import (
+    absolute_import, print_function, unicode_literals, division
+)
+
+from .base import alias, XmlEnumeration, XmlMappedEnumMember
+
+
+@alias('WD_HEADER_TYPE')
+class WD_HEADER_FOOTER_INDEX(XmlEnumeration):
+    """
+    alias: **WD_HEADER_TYPE**
+
+    Specifies the type of a header.
+
+    Example::
+
+        from docx.enum.header import WD_HEADER_TYPE
+
+        header = section.header
+        header.type = WD_HEADER_TYPE.PRIMARY
+    """
+
+    __ms_name__ = 'WdHeaderFooterIndex'
+
+    __url__ = 'https://msdn.microsoft.com/en-us/library/office/ff839314.aspx'
+
+    __members__ = (
+        XmlMappedEnumMember(
+            'PRIMARY', 1, 'default', 'Default/odd header type.'
+        ),
+        XmlMappedEnumMember(
+            'FIRST_PAGE', 2, 'first', 'First page header type.'
+        ),
+        XmlMappedEnumMember(
+            'EVEN_PAGES', 3, 'even', 'Even page header type.'
+        ),
+    )
```

Comment:
Woops! Forgot the docstring. I suspect that's not the last time that's going to happen ;)

Ok cool. Makes sense.